crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1836,6 +1855,15 @@ impl World {
         }
     }
 
+    /// Applies the internal [`CommandQueue`] to self
```

Comment:
```suggestion
    /// Applies any commands in the world's internal [`CommandQueue`].
    ///
    /// This function does nothing if called from inside of a [`Command`] added by a component hook.
```

Correct me if I'm wrong